### ISSUE TITLE
Palmar hyperlinearity

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6464]
**Preferred term label:**
Palmar hyperlinearity

**Synonyms**

**Definition (free text, please give PubMed ID)**
Exaggerated skin markings (dermatoglyphics) on the palms of the hand. PMID:23301728

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Ichthyosis vulgaris:
https://pubmed.ncbi.nlm.nih.gov/23301728/

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
